students/toddschultz/session03/slicing_lab_exercise.py

```diff
@@ -0,0 +1,48 @@
+#with the first and last items exchanged.
+def exchanged_string(seq):
+    return seq[-1:] + seq[1:-1] + seq[:1]
+exchanged_string("abcdefghijklmnopqrstuvwxyz")
```

Comment:
Don't necessarily need to put examples of how to run the functions after each function - although it's nice to see line of thinking.  You should definitely add some tests though, per the assignment.